schemas/actor.json

```diff
@@ -0,0 +1,122 @@
+{
+    "openrpc": "1.2.6",
+    "info": {
+        "title": "Actor",
+        "version": "0.1.0"
+    },
+    "methods": [
+        {
+            "name": "get_parameters",
+            "params": [
+                {
+                    "name": "parameters",
+                    "schema": {
+                            "type": "array",
+                            "items": {
+                                "type": "string"
+                        }
+                    },
+                    "required": true
+                }
+            ],
+            "description": "Get the values for parameters.",
+            "result": {
+                "name": "result",
+                "schema": {
+                    "$ref": "#/components/parameter_value_list"
```

Comment:
Note: This "$ref" points downwards to the components (you asked, what they are for).
That way you can define it once and use it multiple times.

Reply:
Yeah, I've seen that in the spec.